po/cs.po

```diff
@@ -540,46 +539,44 @@ msgid ""
 "UTC time."
 msgstr ""
 "Autentizace je vyžadována pro kontrolu jestli RTC ukládá lokální časovou "
-"zónu nebo UTC čas"
+"zónu nebo UTC čas."
 
 #: ../src/timedate/org.freedesktop.timedate1.policy.in.h:7
 msgid "Turn network time synchronization on or off"
-msgstr "Vypnout nebo zapnout synchronizaci s časem ze sítě "
+msgstr "Zapnout nebo vypnout synchronizaci s časem ze sítě"
 
 #: ../src/timedate/org.freedesktop.timedate1.policy.in.h:8
 msgid ""
 "Authentication is required to control whether network time synchronization "
 "shall be enabled."
-msgstr "Autentizace je požadována pro kontrolu synchronizace času ze sítě"
+msgstr "Autentizace je vyžadována pro kontrolu synchronizace času ze sítě."
 
 #: ../src/core/dbus-unit.c:450
 msgid "Authentication is required to start '$(unit)'."
-msgstr "Autentizace je vyžadována pro spuštění „$(unit)”."
+msgstr "Autentizace je vyžadována pro spuštění '$(unit)'."
 
 #: ../src/core/dbus-unit.c:451
 msgid "Authentication is required to stop '$(unit)'."
-msgstr "Autentizace je vyžadována pro vypnutí „$(unit)”"
+msgstr "Autentizace je vyžadována pro vypnutí '$(unit)'."
 
 #: ../src/core/dbus-unit.c:452
 msgid "Authentication is required to reload '$(unit)'."
-msgstr "Autentizace je vyžadována pro znovu načtení „$(unit)”"
+msgstr "Autentizace je vyžadována pro znovu načtení '$(unit)'."
 
 #: ../src/core/dbus-unit.c:453 ../src/core/dbus-unit.c:454
 msgid "Authentication is required to restart '$(unit)'."
-msgstr "Autentizace je vyžadována pro restart „$(unit)”"
+msgstr "Autentizace je vyžadována pro restart '$(unit)'."
 
 #: ../src/core/dbus-unit.c:560
 msgid "Authentication is required to kill '$(unit)'."
-msgstr "Autentizace je vyžadována pro ukončení „$(unit)”"
+msgstr "Autentizace je vyžadována pro ukončení '$(unit)'."
 
 #: ../src/core/dbus-unit.c:590
 msgid "Authentication is required to reset the \"failed\" state of '$(unit)'."
 msgstr ""
-"Autentizace je vyžadována pro resetování chybného („failed”)  stavu "
-"„$(unit)”."
+"Autentizace je vyžadována pro resetování chybného stavu "
+"'$(unit)'."
```

Comment:
Hm, you're right. I thought I was looking at the combined diff, but it seems either me or github got confused.